Makefile

```diff
@@ -210,3 +219,45 @@ push-image: build-image
 	docker tag "$(OPERATOR_IMAGE):$(OPERATOR_TAG_LONG)" "$(OPERATOR_IMAGE):latest"
 	docker push "$(OPERATOR_IMAGE):$(OPERATOR_TAG_LONG)"
 	docker push "$(OPERATOR_IMAGE):latest"
+
```

Comment:
Add a line like this here:
```
## -- Local deployment and cleanup targets --
```

This will become the section title.

Reply:
Oh ok, as consistency to other targets you merged in your help target PR. Will add.

Reply:
Once this is addressed, we can merge this PR.